CMakeSettings.json

```diff
@@ -2,97 +2,97 @@
   // See https://go.microsoft.com//fwlink//?linkid=834763 for more information about this file.
   "configurations": [
     {
-      "name": "x64-Release",
-      "generator": "Visual Studio 15 2017 Win64",
+      "name": "x86-RelWithDebInfo",
+      "generator": "Visual Studio 15 2017",
       "configurationType": "RelWithDebInfo",
-      "inheritEnvironments": [ "msvc_x64_x64" ],
+      "inheritEnvironments": [ "msvc_x86" ],
       "buildRoot": "${env.USERPROFILE}\\CMakeBuilds\\${workspaceHash}\\build\\${name}",
       "installRoot": "${env.USERPROFILE}\\CMakeBuilds\\${workspaceHash}\\install\\${name}",
       "cmakeCommandArgs": "",
-      "buildCommandArgs": "-m -v:minimal",
+      "buildCommandArgs": "",
       "variables": [
         {
-          "name": "OPENSSL_OFF",
+          "name": "INSTALLER_MERGE_MODULES",
           "value": "OFF"
         },
         {
-          "name": "CMAKE_BUILD_TYPE",
+          "name": "ENABLE_SQL",
+          "value": "OFF"
+        },
+        {
+          "name": "CMAKE_BUILD_TYPE_INIT",
           "value": "RelWithDebInfo"
         },
         {
-          "name": "USE_SYSTEM_OPENSSL",
-          "value": "OFF"
+          "name": "WIN32",
+          "value": "WIN32"
         },
         {
-          "name": "PORTABLE",
-          "value": "ON"
+          "name": "ENABLE_LIBRDKAFKA",
+          "value": "OFF"
         },
         {
-          "name": "USE_SYSTEM_ZLIB",
+          "name": "ENABLE_JNI",
           "value": "OFF"
         },
         {
-          "name": "DISABLE_CONTROLLER",
-          "value": "TRUE"
+          "name": "OPENSSL_OFF",
+          "value": "OFF"
         },
         {
-          "name": "USE_SYSTEM_UUID",
+          "name": "ENABLE_COAP",
           "value": "OFF"
         },
         {
           "name": "USE_SHARED_LIBS",
           "value": "OFF"
         },
         {
-          "name": "FORCE_WINDOWS",
+          "name": "DISABLE_CONTROLLER",
           "value": "ON"
         },
         {
-          "name": "DISABLE_ROCKSDB",
-          "value": "FALSE"
-        },
-        {
-          "name": "FAIL_ON_WARNINGS",
-          "value": "OFF"
-        },
-        {
-          "name": "ENABLE_WEL",
-          "value": "TRUE"
+          "name": "BUILD_ROCKSDB",
+          "value": "ON"
         },
         {
-          "name": "ENABLE_SQL",
-          "value": "TRUE"
+          "name": "FORCE_WINDOWS",
+          "value": "ON"
         },
         {
-          "name": "ENABLE_LIBRDKAFKA",
+          "name": "USE_SYSTEM_UUID",
           "value": "OFF"
         },
         {
-          "name": "DISABLE_CURL",
```

Comment:
What happened to this flag?

Reply:
It is not present in `win_build_vs.bat` and @fgerlits changed this file to be roughly equivalent to `win_build_vs.bat /D`